homeassistant/components/media_player/liveboxplaytv.py

```diff
@@ -76,19 +77,32 @@ def __init__(self, host, port, name):
         self._channel_list = {}
         self._current_channel = None
         self._current_program = None
+        self._media_duration = None
+        self._media_remaining_time = None
+        self._media_ = None
         self._media_image_url = None
+        self._media_last_updated = None
 
     @asyncio.coroutine
     def async_update(self):
         """Retrieve the latest data."""
+        import pyteleloisirs
```

Comment:
We should either expose `get_program_duration` and `get_remaining_time` off of liveboxplaytv, or specifically call out `pyteleloisirs` in the requirements list since we're using it directly. We shouldn't implicitly depend on liveboxplaytv to pull in the requirement if we're calling it directly.